.mk/k8s.mk

```diff
@@ -35,11 +35,6 @@ include .mk/packet.mk
 include .mk/aws.mk
 include .mk/azure.mk
 
-# .kind.mk enables the kind.sigs.k8s.io docker based K8s install:
```

Comment:
@lobkovilya could you please rebase your MR, I've merged MR from @nickolaev kubernetes upgrade, and it looks it was this changes already. Thanks.